user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserQuotasOperator.java

```diff
@@ -24,7 +23,7 @@
 import java.util.Set;
 
 public class KafkaUserQuotasOperator {
-    private static final Logger log = LogManager.getLogger(KafkaUserQuotasOperator.class.getName());
+    private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(KafkaUserQuotasOperator.class.getName());
```

Comment:
Looks like a regular LOGGER would be sufficient? But it is quite weird since this has to be tried to a reconciliation. So I think you need to pass `Reconciliation` in here and use it.